### PR TITLE
MM-44652 - Fix long team name

### PR DESCRIPTION
**Pull request by avinashlng1080**
#### Summary
Wraps long team names so as not to push the  "+ icon " next to it.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48169

#### Checklist
- [X] Has UI changes

#### Screenshots
fix-long-team-name

#### Release Note
```release-note
Wraps long team name so as not to push the  "+ icon " next to it.
```

### CODE REVIEW

File: app/screens/home/channel_list/categories_list/header/header.tsx [hunk 1]
```diff
@@ -88,6 +88,13 @@ const getStyles = makeStyleSheetFromTheme((theme: Theme) => ({
         justifyContent: 'space-between',
         height: 40,
     },
+    outsideBox: {
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+    },
+    firstBox: {
+        width: '85%',
```
Comment (enahum) [inline app/screens/home/channel_list/categories_list/header/header.tsx:96]: Why 85%?

**Review by larkox — APPROVED** [review]
LGTM, just wanted to confirm a few things:

- The plus button seems to me a bit out of place. Not sure if because the two lines of team name, but I would prefer it a bit more centered. But that is something I will defer to UX.
- The way it is organized now, the subheading will only go as far as the team name (the plus button will always cover the whole vertical of the heading and subheading. Is this intended? How does this look with a long server display name?

**Comment by matthewbirtch** [comment]
> * The plus button seems to me a bit out of place. Not sure if because the two lines of team name, but I would prefer it a bit more centered. But that is something I will defer to UX.

You're right @larkox. The plus button is misaligned. It's possible that has been that way for a while. It looks like it has been pushed up a bit compared to the main branch. The plus button should be vertically aligned with the first line of the team name and with the server icon on the left side (see the red line in the screenshot below.

image

> * The way it is organized now, the subheading will only go as far as the team name (the plus button will always cover the whole vertical of the heading and subheading. Is this intended? How does this look with a long server display name?

Long server display names should be truncated to one line (which I believe it already is).

**Comment by avinashlng1080** [comment]
I corrected the margin issue.


corrected

File: app/screens/home/channel_list/categories_list/header/header.tsx (hunk 1 above)
Comment (avinashlng1080) [reply app/screens/home/channel_list/categories_list/header/header.tsx:96]: I worked out this ratio value from the designs... it should have been 88% but 85% looks better.

**Comment by avinashlng1080** [comment]
@matthewbirtch please see update

**Review by enahum — APPROVED** [review]

**Review by matthewbirtch — APPROVED** [review]
Approving based on the screenshots - I wasn't able to get it running locally

**Comment by matthewbirtch** [comment]
> @matthewbirtch please see update

@avinashlng1080 did your changes get pushed? I got it working locally now, but I don't see the change and I don't see a commit with this change in the github PR history above

**Comment by avinashlng1080** [comment]
@matthewbirtch  it didn't get pushed yesterday.  My bad.  Can you please check again?

**Comment by matthewbirtch** [comment]
Looks good @avinashlng1080. Thanks!

**Pull request merged**